FiveCalls/FiveCalls/Extensions/UIColor+FiveCalls.swift

```diff
@@ -37,9 +37,7 @@ extension UIColor {
     //MARK - Grays
     
     convenience init(fvc_gray component: Float) {
-        self.init(colorLiteralRed: component,
-                  green: component,
-                  blue: component,
+        self.init(white: CGFloat(component),
                   alpha: 1.00)
```

Comment:
Why not just get rid of this convenience init altogether and inline the `white:` calls?  We'll get the CGFloat conversion for free there anyway.